test/system/helpers.bash

```diff
@@ -1,3 +1,4 @@
+#!/usr/bin/env bats
```

Comment:
IIRC @edsantiago said doing this is bad-juju since it implies the file is a script that people should execute.  Is there a way to block the bash lint-checking for this file?

Reply:
My impression was that that file was a bats test file but I am more than happy to use another approach. Maybe we could change its extension? I would prefer to avoid excludes because they are hard to maintain and it would make impossible to run bashate directly, without pre-commit.

Reply:
My understanding is this file is special.  We really need @edsantiago input here since he knows bats the best.